.cspell-partial.json

```diff
@@ -1,9 +1,5 @@
 {
-  "ignorePaths": [
-    "**/perception/**",
-    "sensing/tier4_pcl_extensions/include/**",
-    "perception/bytetrack/lib/**"
-  ],
+  "ignorePaths": ["sensing/tier4_pcl_extensions/include/**", "perception/bytetrack/lib/**"],
```

Comment:
Does this PR intend to fix this file?

Reply:
Yes, it is intended.
This PR is the very last PR for removing the spell-check errors from perception directory (and also for the whole Autoware repository) except some inevitable packages like bytetrack and tier4_pcl_extensions which are related to external packages and cannot be modified easily.

I just wanted to include it so that we can actually check that the whole spell-check errors have been removed by simply checking the `spell-check-partial` CI in this PR. But not a strong reason, so can make a separate PR if necessary.